script/Ciel/Day 6/06_01_CIEL06_6.txt

```diff
@@ -390,35 +415,41 @@ Here, we will be able to focus on the experiment without interruption.
 [sha:4db0f4ce0632bd8cf531d5333b3a6043162b12a2]{
 -- Page 250, Offset 13818.
 -- 何者であれ―――たとえ星の意識であれ、この彷徨海を見つける事はできない。違いますか？』
-No one, no matter who―not even the consciousness of the world―will be able to find this Wandering Sea. Am I wrong?'
+// 星の意識 seems unique in the TL - lore term or not?
+No-one―――not even the consciousness of the planet―――will be able to find this Wandering Sea. Is that right?'
 }
 [sha:48f27ed555189b638a60939b53868b39e5031e06]{
 -- Page 251, Offset 13819.
 -- 『……それも次の新年までだがな。
-'...Until next year, yes.
+'...Until the advent of the new year, yes.
 }
 [sha:d3bca31aa87ab2ca5e52b75f343f978af7771875]{
 -- Page 251, Offset 13820.
 -- いいだろう。余命一年の同胞の依頼だ、喜んで受けよう。
-Very well. I gladly accept the request of my brethren, who only has one year left to live.
+Very well. As you are my brethren, and seeing as how you have but one year left to live, I'll accept your request.
 }
 [sha:6f1a7e90ed5b16f3e59cd29d2643d5d58db12e68]{
 -- Page 251, Offset 13821.
 -- 君が北海で何を企てたか、いささか興味もある。貴重な逸話として拝聴したい』
-I am slightly curious as to what you've been plotting in the North Sea. I would gladly lend an ear to your precious anecdotes.'
+I admit that I am also a little curious about what you've been up to in the North Sea.
+ I would gladly lend and ear to any precious anecdotes you have to share.'
 }
 [sha:914181d5f7183257913a49c3f697ffc6af29ad44]{
 -- Page 252, Offset 13822.
 -- 『いやあ、その件については本題の後に。そう面白い話にもなりませんでしたしね。
-'Oh well, that will have to wait until after the main topic. It isn't particularly interesting anyway.'
+'Ah, I'm afraid that will have to wait until after the main event.
+ It isn't particularly interesting anyway.
 }
 [sha:4c23e3d58ec835120e0227fd94aa438bf6c48124]{
 -- Page 252, Offset 13823.
 -- さて。それでは貴方の中の原理を、わずかですが現実に<零|こぼ>す神秘を教授いたしましょう。
-Now then. If I may instruct you on a Mystery that shall allow you, if only very slightly, to seep out the Principle contained inside of you.
+Now then.
```

Comment:
Now then. Allow me to instruct you an arcanum for spilling—if only slightly—the Principle you carry within you into reality.